### ISSUE TITLE
Move to no_std?

### ISSUE THREAD

**Issue by Qix-** [JiatLn/color-art#2]
Hi there,

Is there any chance of transitioning this to a `no_std` crate? I haven't scoured the sources enough to see whether or not this would be a trivial change, but I would imagine at least the core color structures and model conversion algorithms would be `no_std`-able at the very least.

Would be happy to put in a PR if it is given the green light.

Thanks!

**Comment by JiatLn** [comment]
Certainly! I think that is a good idea. I'm glad you can submit a pull request。 @Qix-
